Gemfile

```diff
@@ -26,6 +26,10 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'pg', '~> 0.18.4'
 # Devise for user and authentication handling
 gem 'devise', '~> 3.5.1'
+# Active Admin and its dependencies
+gem 'activeadmin', github: 'activeadmin'
```

Comment:
Better use `~> 1.0.0.pre4`, because you are only indicating the GitHub where it is located.